reference/sectionC_interpretation.inc

```diff
@@ -398,8 +398,8 @@ Interpretation of map_variables
 
 .. _specC_interpretation_of_variable_resets:
 
-Interpretation of variable resets
----------------------------------
+Interpretation of ``variable`` ``reset`` elements
```

Comment:
I think this should be
```
Interpretation of ``reset`` elements
```

Reply:
I agree ... but it used to be "Interpretation of resets" ... then we added the "variable" quite recently? Not sure why though ...

Reply:
Done in 3bf4167